### PR TITLE
Fiscalcode wizard split

### PR DESCRIPTION
**Pull request by rds-alfredosalata**
1. Split fiscalcode module in two modules to split functionality. Wizard computation might be unwanted by certain users.
2. Split tests.
3. Python library codicefiscale is now only required for calculation module and it's optional for base functionality.
4. Added Nominal Company company type which  allows for better dynamic computation of CF requirements.

### CODE REVIEW

File: l10n_it_fiscalcode/README.rst [hunk 1]
```diff
@@ -1,96 +1,65 @@
...
-   :target: https://odoo-community.org
-
-OCA, or the Odoo Community Association, is a nonprofit organization whose
-mission is to support the collaborative development of Odoo features and
-promote its widespread use.
-
-This module is part of the `OCA/l10n-italy <https://github.com/OCA/l10n-italy/tree/12.0/l10n_it_fiscalcode>`_ project on GitHub.
-
-You are welcome to contribute. To learn how please visit https://odoo-community.org/page/Contribute.
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (eLBati) [inline l10n_it_fiscalcode/README.rst]: Do not replace the dynamic generated README
See https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: l10n_it_fiscalcode/__manifest__.py [hunk 2]
```diff
@@ -1,30 +1,23 @@
...
-    'data': [
-        "security/ir.model.access.csv",
-        'data/res.city.it.code.csv',
-        'view/fiscalcode_view.xml',
-        'view/report_invoice_document.xml',
-        'wizard/compute_fc_view.xml',
-        ],
-    'installable': True
-}
+# -*- coding: utf-8 -*-
```
Comment (eLBati) [inline l10n_it_fiscalcode/__manifest__.py]: Do not add back `# -*- coding: utf-8 -*-`

File: l10n_it_fiscalcode/__manifest__.py [hunk 3]
```diff
@@ -1,30 +1,23 @@
...
+# Copyright 2016 Andrea Gallina (Apulia Software)
+# Copyright 2018 Matteo Bilotta (Link IT s.r.l.)
+# Copyright 2018 Lorenzo Battistini (https://github.com/eLBati)
+# Copyright 2018 Teuron (<http://www.teuron.it>)
+# Copyright 2018 RDS S.p.A. (<http://www.rdsmoulding.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Italian Localisation - Fiscal Code',
+    'version': '12.0.0.0.0',
```
Comment (eLBati) [inline l10n_it_fiscalcode/__manifest__.py]: Version should probably be `12.0.2.0.0`.
See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers

File: l10n_it_fiscalcode/__manifest__.py [hunk 4]
```diff
@@ -1,30 +1,23 @@
...
+    'category': 'Localisation/Italy',
+    'author': "Teuron S.r.l.",
```
Comment (eLBati) [inline l10n_it_fiscalcode/__manifest__.py]: If you want to remove original authors, you must at least keep `Odoo Community Association (OCA)`.
See https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py

File: l10n_it_fiscalcode/__manifest__.py [hunk 5]
```diff
@@ -1,30 +1,23 @@
...
+    'website': 'http://www.teuron.it',
```
Comment (eLBati) [inline l10n_it_fiscalcode/__manifest__.py]: Website must be https://github.com/OCA/l10n-italy or https://github.com/OCA/l10n-italy/tree/12.0/l10n_it_fiscalcode

File: l10n_it_fiscalcode/i18n/it.po [hunk 6]
```diff
@@ -1,262 +1,76 @@
 # Translation of Odoo Server.
```
Comment (eLBati) [inline l10n_it_fiscalcode/i18n/it.po]: Translaiotns must be done by weblate.
See https://odoo-community.org/blog/the-oca-blog-1/post/translations-platform-81

File: l10n_it_fiscalcode/README.rst (hunk 1 above)
Comment (eLBati) [reply l10n_it_fiscalcode/README.rst]: I still see file replaced

**Comment by eLBati** [comment]
@rds-alfredosalata nel diff ci sono ancora modifiche ai file .po.
Potresti provare a fare una PR , idealmente con un solo commit, dove modifichi puntualmente solo i file che ti interessano?

**Comment by SimoRubi** [comment]
@rds-alfredosalata pensi di riuscire a portare avanti questa PR o possiamo chiudere?

**Comment by OpenCode** [comment]
> @rds-alfredosalata pensi di riuscire a portare avanti questa PR o possiamo chiudere?

Propongo di chiudere. Visto il tempo che è trascorso e la versione coinvolta. Sarà poi nell'interesse di qualcuno riaprirla se davvero necessaria.

**Pull request closed without merging**